cmd/dlt/kubeletconfig/cmd.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeletconfig
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "kubeletconfig",
+	Aliases: []string{"kubelet-config"},
+	Short:   "Delete the custom kubeletconfig for a cluster",
+	Long:    "Delete the custom kubeletconfig for a cluster",
+	Example: `  # Delete the custom kubeletconfig for cluster 'foo'
```

Comment:
is this space necessary

Reply:
@den-rgb It's to do with the formatting of the output when you use the `rosa help` command e.g `rosa help delete kubeletconfig`. All other commands have the spaces, so I was just being consistent.